pilot/pkg/model/context.go

```diff
@@ -273,6 +273,15 @@ type Proxy struct {
 	// NOTE: DO NOT USE THIS FIELD TO CONSTRUCT DNS NAMES
 	ConfigNamespace string
 
+	// Labels specifies the set of workload instance (ex: k8s pod) labels associated with this node.
+	// Labels can be different from that in Metadata because of pod labels update after startup,
+	// while NodeMetadata.Labels are set during bootstrap.
+	Labels map[string]string `json:"LABELS,omitempty"`
```

Comment:
```suggestion
	Labels map[string]string
```

Do we need `json` stuff? It seems confusing since this isn't actually marshaled to/from json

Reply:
seems no